exporters/elasticsearch-exporter/src/test/java/io/zeebe/exporter/ElasticsearchExporterIT.java

```diff
@@ -184,34 +106,34 @@ private void assertRecordExported(Record<?> record) {
     assertThat(source).isEqualTo(recordToMap(record));
   }
 
-  protected ElasticsearchTestClient createElasticsearchClient() {
+  protected ElasticsearchTestClient createElasticsearchClient(
+      ElasticsearchExporterConfiguration configuration) {
     return new ElasticsearchTestClient(
-        getConfiguration(), new ZbLogger("io.zeebe.exporter.elasticsearch"));
-  }
-
-  private ElasticsearchExporterConfiguration getConfiguration() {
-    final ExporterCfg cfg = getExporterCfg();
-    if (cfg.getArgs() != null) {
-      return deserialize(cfg.getArgs(), ElasticsearchExporterConfiguration.class);
-    } else {
-      return new ElasticsearchExporterConfiguration();
-    }
-  }
-
-  private ExporterCfg getExporterCfg() {
-    return testRule
-        .getBrokerCfg()
-        .getExporters()
-        .stream()
-        .filter(cfg -> cfg.getId().equals("elasticsearch"))
-        .findFirst()
-        .orElseThrow(
-            () ->
-                new AssertionError("Failed to find configuration for exporter " + "elasticsearch"));
+        configuration, new ZbLogger("io.zeebe.exporter.elasticsearch"));
   }
 
-  private <T> T deserialize(final Map<String, Object> value, final Class<T> valueClass) {
-    return MAPPER.convertValue(value, valueClass);
+  private ElasticsearchExporterConfiguration newConfiguration() {
```

Comment:
Sorry it seems some of my review comments from yesterday got lost :see_no_evil: 

I also would propose a method in the exporter it rule to get the exporter configuration from the BrokerCfg something like this:

```java
public <T> T getExporterConfiguration(Class<?> exporterClass, Class<T> configurationClass) {
  return brokerRule
      .getBrokerCfg()
      .getExporters()
      .stream()
      .filter(cfg -> cfg.getClassName().equals(exporterClass.getCanonicalName()))
      .findFirst()
      .map(cfg -> new ObjectMapper().convertValue(cfg.getArgs(), configurationClass))
      .orElseThrow(
          () ->
              new IllegalArgumentException(
                  "No exporter for class " + exporterClass.getCanonicalName() + " configured"));
}
```

This would allow us to reduce the elastic IT to, as the exporter is already configured in the `zeebe.test.cfg.toml`:

```java
@Rule public ExporterIntegrationRule exporterBrokerRule = new ExporterIntegrationRule();

private ElasticsearchTestClient esClient;

@Before
public void setUp() {
  esClient =
      createElasticsearchClient(
          exporterBrokerRule.getExporterConfiguration(
              ElasticsearchExporter.class, ElasticsearchExporterConfiguration.class));
}

@After
public void tearDown() throws IOException {
  if (esClient != null) {
    esClient.close();
    esClient = null;
  }
}

@Test
public void shouldExportRecords() {
  // when
  exporterBrokerRule.performSampleWorkload();

  // then

  // assert index settings for all created indices
  assertIndexSettings();

  // assert all records which where recorded during the tests where exported
  exporterBrokerRule.visitExportedRecords(this::assertRecordExported);
}
```